tests/CMakeLists.txt

```diff
@@ -178,7 +178,7 @@ add_test(NAME test_build_all_off
 
 add_test(NAME test_build_all_on
         COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options.sh
-        ${PROJECT_SOURCE_DIR} test_build_all_on "-DBUILD_WITH_CODE_COVERAGE=ON -DBUILD_OSTREE=ON -DBUILD_DEB=ON")
+        ${PROJECT_SOURCE_DIR} test_build_all_on "-DBUILD_WITH_CODE_COVERAGE=ON -DBUILD_OSTREE=ON -DBUILD_DEB=ON -DBUILD_SOTA_TOOLS=ON -DBUILD_ISOTP=ON -DBUILD_OPCUA=ON -DBUILD_LOAD_TESTS=ON")
```

Comment:
`BUILD_SYSTEMD`?

Reply:
`BUILD_SYSTEMD` is on the default. I could add it though?

Reply:
Oh that's right. Porbably don't need to worry about it then.